### ISSUE TITLE
need help to install antspyx==0.2.4

### ISSUE THREAD

**Issue by flaskdoris** [ANTsX/ANTsPy#531]
**Describe the bug**
I have problems with installing antspyx==0.2.4, im using vs code and have created a virtual environment using python version 3.8 and try to install antspyx==0.2.4 by the command pip install 'antspyx @ git+https://github.com/ANTsX/ANTsPy@v0.2.4' but it is not working. This is the error I get: 

(venv) PS C:\Users\torao\Desktop\MEX> pip install 'antspyx @ git+https://github.com/ANTsX/ANTsPy@v0.2.4'
Collecting antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4
  Cloning https://github.com/ANTsX/ANTsPy (to revision v0.2.4) to c:\users\torao\appdata\local\temp\pip-install-3fe63gxr\antspyx_3cd6c5ede66f41acb1612628fedbd262
  Running command git clone --filter=blob:none --quiet https://github.com/ANTsX/ANTsPy 'C:\Users\torao\AppData\Local\Temp\pip-install-3fe63gxr\antspyx_3cd6c5ede66f41acb1612628fedbd262'
  Running command git checkout -q 6ec0f9ba8c2ea45a331288314f5c20d4331a67a7
  Resolved https://github.com/ANTsX/ANTsPy to commit 6ec0f9ba8c2ea45a331288314f5c20d4331a67a7
  Preparing metadata (setup.py) ... done
Requirement already satisfied: pandas in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.0.1)
Requirement already satisfied: numpy in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.18.2)
Requirement already satisfied: scipy in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.10.1)
Requirement already satisfied: sklearn in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.0.post12)
Requirement already satisfied: scikit-image in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.21.0)
Requirement already satisfied: scikit-learn in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.3.2)
Requirement already satisfied: statsmodels in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.14.1)
Requirement already satisfied: webcolors in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.13)
Requirement already satisfied: matplotlib in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.7.4)
Requirement already satisfied: pyyaml in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (6.0.1)
Requirement already satisfied: chart_studio in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.1.0)
Requirement already satisfied: Pillow in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (10.2.0)
Requirement already satisfied: nibabel in c:\users\torao\desktop\mex\venv\lib\site-packages (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (5.2.0)
Requirement already satisfied: plotly in c:\users\torao\desktop\mex\venv\lib\site-packages (from chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (5.18.0)
Requirement already satisfied: requests in c:\users\torao\desktop\mex\venv\lib\site-packages (from chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2.31.0)
Requirement already satisfied: retrying>=1.3.3 in c:\users\torao\desktop\mex\venv\lib\site-packages (from chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.3.4)        
Requirement already satisfied: six in c:\users\torao\desktop\mex\venv\lib\site-packages (from chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.16.0)
Requirement already satisfied: contourpy>=1.0.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.1.1)
Requirement already satisfied: cycler>=0.10 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.12.1)
Requirement already satisfied: fonttools>=4.22.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (4.47.2)       
Requirement already satisfied: kiwisolver>=1.0.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.4.5)        
Collecting numpy (from antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4)
  Using cached numpy-1.24.4-cp38-cp38-win_amd64.whl.metadata (5.6 kB)
Requirement already satisfied: packaging>=20.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (23.2)
Requirement already satisfied: pyparsing>=2.3.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.1.1)
Requirement already satisfied: python-dateutil>=2.7 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2.8.2)     
Requirement already satisfied: importlib-resources>=3.2.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (6.1.1)
Requirement already satisfied: pytz>=2017.2 in c:\users\torao\desktop\mex\venv\lib\site-packages (from pandas->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2024.1)
Requirement already satisfied: networkx>=2.8 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-image->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.1)
Requirement already satisfied: imageio>=2.27 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-image->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2.33.1)
Requirement already satisfied: tifffile>=2022.8.12 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-image->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2023.7.10)
Requirement already satisfied: PyWavelets>=1.1.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-image->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.4.1)      
Requirement already satisfied: lazy_loader>=0.2 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-image->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.3)
Requirement already satisfied: joblib>=1.1.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-learn->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (1.3.2)
Requirement already satisfied: threadpoolctl>=2.0.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from scikit-learn->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.2.0)   
Requirement already satisfied: patsy>=0.5.4 in c:\users\torao\desktop\mex\venv\lib\site-packages (from statsmodels->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (0.5.6)
Requirement already satisfied: zipp>=3.1.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from importlib-resources>=3.2.0->matplotlib->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.17.0)
Requirement already satisfied: tenacity>=6.2.0 in c:\users\torao\desktop\mex\venv\lib\site-packages (from plotly->chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (8.2.3)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\torao\desktop\mex\venv\lib\site-packages (from requests->chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in c:\users\torao\desktop\mex\venv\lib\site-packages (from requests->chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (3.6)   
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\torao\desktop\mex\venv\lib\site-packages (from requests->chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2.2.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\torao\desktop\mex\venv\lib\site-packages (from requests->chart_studio->antspyx@ git+https://github.com/ANTsX/ANTsPy@v0.2.4) (2024.2.2)
Using cached numpy-1.24.4-cp38-cp38-win_amd64.whl (14.9 MB)
Building wheels for collected packages: antspyx
  Building wheel for antspyx (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      running bdist_wheel
      running build
      running build_py
      running build_ext
      error: [WinError 193] %1 är inte ett giltigt Win32-program
      Configuring ITK
      No local ITK installation found... Building ITK now...
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for antspyx
  Running setup.py clean for antspyx
Failed to build antspyx
ERROR: Could not build wheels for antspyx, which is required to install pyproject.toml-based projects
(venv) PS C:\Users\torao\Desktop\MEX>

**Comment by cookpa** [comment]
ANTsPy [v0.3.8](https://github.com/ANTsX/ANTsPy/releases/tag/v0.3.8) is the first that will work on Windows. If you need to build 0.2.4, you'll need to build for another OS
